.jenkins

```diff
@@ -174,7 +197,6 @@ pipeline {
                         }
                     }
                 }
-/*
```

Comment:
It seems that this step is now passing, too, but I don't think activating this build again belongs here.

Reply:
Nevermind, I didn't read this corrcetly.